### PR TITLE
Add OMSO

### PR DESCRIPTION
**Pull request by emileadipura**

### CODE REVIEW

**Comment by emileadipura** [comment]
Check @JeremyParish69

File: neutron/assetlist.json [hunk 1]
```diff
@@ -891,6 +891,34 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/badkid.png"
       }
+    },
+    {
+      "description": "Astro BOY",
```
Comment (JeremyParish69) [inline neutron/assetlist.json:896]: This has already been added a few lines up. Please remove this duplicate.

File: osmosis/assetlist.json [hunk 2]
```diff
@@ -14095,31 +14095,31 @@
       }
     },
     {
-      "description": "A clan of 11y bad kids crafting chaos on the Cosmos eco. One bad memecoin to rule them all  $BADKID. Airdropped to Badkids NFT holders and $STARS stakers. It's so bad, your wallet's throwing a tantrum for it.",
+      "description": "Created as a culmination of all the memeable events on osmosis.",
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:14098]: Please don't overwrite previously added assets.

**Review by JeremyParish69 — CHANGES_REQUESTED** [review]
Please don't change anything else in the file. Don't remove the original BOY and don't delete BADKID.

**Pull request closed without merging**